src/main/java/application/rest/StackUtils.java

```diff
@@ -0,0 +1,547 @@
+package application.rest;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.HttpClients;
+import org.yaml.snakeyaml.Yaml;
+
+import io.kabanero.v1alpha2.client.apis.KabaneroApi;
+import io.kabanero.v1alpha2.models.Stack;
+import io.kabanero.v1alpha2.models.StackList;
+import io.kabanero.v1alpha2.models.StackSpec;
+import io.kabanero.v1alpha2.models.StackSpecImages;
+import io.kabanero.v1alpha2.models.StackSpecPipelines;
+import io.kabanero.v1alpha2.models.StackSpecVersions;
+import io.kabanero.v1alpha2.models.StackStatus;
+import io.kabanero.v1alpha2.models.StackStatusVersions;
+import io.kabanero.v1alpha2.models.Kabanero;
+import io.kabanero.v1alpha2.models.KabaneroList;
+import io.kabanero.v1alpha2.models.KabaneroSpecStacks;
+import io.kabanero.v1alpha2.models.KabaneroSpecStacksRepositories;
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.apis.CoreV1Api;
+import io.kubernetes.client.models.V1ContainerStatus;
+import io.kubernetes.client.models.V1Pod;
+import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.models.V1PodStatus;
+import io.kubernetes.client.proto.Meta.Status;
+
+public class StackUtils {
+	
+	public static boolean readGitSuccess=true;
```

Comment:
should this initially be false ?

Reply:
Dave will consider and probably change in upcoming version